flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/context/SessionContextTest.java

```diff
@@ -64,15 +64,12 @@ public static void closeResources() {
 
     @Test
     public void testSetAndResetOption() {
-        // table config option
-        sessionContext.set(TABLE_SQL_DIALECT.key(), "hive");
```

Comment:
Why remove it?

Reply:
Currently the SessionContext  try to load the hive parser when setting the dialect hive, but gateway doesn't rely on the hive. So I remove this.